packages/in_app_purchase/example/ios/Podfile

```diff
@@ -0,0 +1,63 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
Please remove `Podfile` from repo. We want to allow Flutter tooling to regenerate it, to ensure the plugin works with the newest version.